pysmt/fnode.py

```diff
@@ -816,6 +816,12 @@ def __rmul__(self, right):
     def __div__(self, right):
         return self._apply_infix(right, _mgr().Div, _mgr().BVUDiv)
 
+    def __floordiv__(self, right):
+        return self._apply_infix(right, _mgr().FloorDiv, None)
```

Comment:
The none for BV is going to raise a very ugly exception. _apply_infix expects two functions, one for the Rational/Integer case and one for the Bitvector. If this does not apply to BV, we should raise a proper exception.